indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -64,17 +64,17 @@
     background-color: lighten($dark-gray, $color-variation);
     cursor: move;
 
-    &:before,
-    & > .handle:before {
+    &::before,
+    & > .handle::before {
```

Comment:
Nice! I updated the wiki to always use `::` instead of `:` as suggested in [MDN], since `:` is a CSS2 syntax and `::` is CSS3 (https://developer.mozilla.org/en/docs/Web/CSS/Pseudo-elements)